plugin/src/test/java/org/jenkinsci/plugins/structs/describable/DescribableModelTest.java

```diff
@@ -73,16 +76,27 @@
     public static JenkinsRule rule = new JenkinsRule();
     @ClassRule
     public static LoggerRule logging = new LoggerRule().record(DescribableModel.class, Level.ALL);
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
 
     @Test
     public void instantiate() throws Exception {
-        Map<String,Object> args = map("text", "hello", "flag", true, "ignored", "!");
+        Map<String,Object> args = map("text", "hello", "flag", true);
         assertEquals("C:hello/true", instantiate(C.class, args).toString());
         args.put("value", "main");
         assertEquals("I:main/hello/true", instantiate(I.class, args).toString());
         assertEquals("C:goodbye/false", instantiate(C.class, map("text", "goodbye")).toString());
     }
 
+    @Test
+    public void erroneousParameters() throws Exception {
+        exceptionRule.expect(IllegalArgumentException.class);
+        exceptionRule.expectMessage(is("Unknown parameter(s) found for class type '" +
```

Comment:
nit: FWIW, It usually feels more readable to me to just use `try/catch` with assertions on the exception directly in the test vs stuff like `ExpectedException`, something like this:

```
try {
    Map<String,Object> args = map("text", "hello", "flag", true, "garbage", "!", "junk", "splat");
    instantiate(C.class,  args);
    Assert.fail("Instantiation should have failed due to unnecessary arguments")
} catch (Exception e) {
    Assert.assertThat(e, Matchers.instanceOf(IllegalArgumentException.class));
    ...
}
```

Reply:
No longer throwing an exception.  Will check logs for appropriate warning.